Explorer/Assets/DCL/Character/CharacterObject/WorldExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using Arch.Core;
+using DCL.Character.Components;
+using ECS.Abstract;
+
+namespace DCL.Character
+{
+    public static class WorldExtensions
+    {
+        private static readonly QueryDescription QUERY = new QueryDescription().WithAll<PlayerComponent>();
+
+        public static SingleInstanceEntity CachePlayer(this World world) =>
```

Comment:
We have `PersistentEntities.Player` already. Can we unify how we use it? Either `PersistentEntities` or extensions: I believe there should be only one way left

Reply:
`PersistentEntities.Player` is the entity representation for the Player in each ECS scene, this one is the global player entity in which only one exists

Reply:
Yeah, true, it's just not obvious